run.sh

```
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+docker run -v $(pwd)/persistence:/root/persistence -p 8080:8080 \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)